From 4dc71711f455f18c9ee41818d1a5ca69512f058f Mon Sep 17 00:00:00 2001
From: Ondrej Jirman <megous@megous.com>
Date: Thu, 6 Feb 2020 04:58:32 +0100
Subject: [PATCH 199/478] arm64: dts: sun50i-a64-pinephone: Shorten
 post-power-on-delay on mmcs

These are not needed.

Signed-off-by: Ondrej Jirman <megous@megous.com>
---
 arch/arm64/boot/dts/allwinner/sun50i-a64-pinephone.dtsi | 3 +++
 1 file changed, 3 insertions(+)

diff --git a/arch/arm64/boot/dts/allwinner/sun50i-a64-pinephone.dtsi b/arch/arm64/boot/dts/allwinner/sun50i-a64-pinephone.dtsi
index 478b6c80b0e0..fe48d3dc1c70 100644
--- a/arch/arm64/boot/dts/allwinner/sun50i-a64-pinephone.dtsi
+++ b/arch/arm64/boot/dts/allwinner/sun50i-a64-pinephone.dtsi
@@ -541,6 +541,7 @@ &mmc0 {
 	cd-gpios = <&pio 5 6 GPIO_ACTIVE_LOW>; /* PF6 */
 	disable-wp;
 	bus-width = <4>;
+	post-power-on-delay-ms = <1>; /* power is already turned on by the bootloader */
 	status = "okay";
 };
 
@@ -551,6 +552,7 @@ &mmc1 {
 	vqmmc-supply = <&reg_dldo4>;
 	bus-width = <4>;
 	non-removable;
+	post-power-on-delay-ms = <1>; /* wifi power is always on */
 	status = "okay";
 
 	rtl8723cs: wifi@1 {
@@ -566,6 +568,7 @@ &mmc2 {
 	bus-width = <8>;
 	non-removable;
 	cap-mmc-hw-reset;
+	post-power-on-delay-ms = <1>; /* power is already turned on by the bootloader */
 	status = "okay";
 };
 
-- 
2.35.3

